### ISSUE TITLE
Why pack of Microsoft.Msagl.WpfGraphControl.nuspec is commented

### ISSUE THREAD

**Issue by idormenco** [Microsoft/automatic-graph-layout#171]
I want to use GraphViewer in a WPF project but i can't find any nuget that can help me.
Please support .

**Comment by Michael-F-Bryan** [comment]
I was wondering the same thing. It'd be nice if the WPF viewer was available on nuget, but it seems like @saveenr removed it in #168.

```diff
@@ -4,5 +4,5 @@ Push-Location $PSScriptRoot
  nuget pack Microsoft.Msagl.nuspec
  nuget pack Microsoft.Msagl.Drawing.nuspec
  nuget pack Microsoft.Msagl.GraphViewerGDI.nuspec
- nuget pack Microsoft.Msagl.WpfGraphControl.nuspec
+ #nuget pack Microsoft.Msagl.WpfGraphControl.nuspec
```

**Comment by saveenr** [comment]
We are looking into publishing a separate NuGet package for the WPF viewer

**Comment by ArminGeisbusch** [comment]
A WPF Viewer is still missing as a nuget package. When could we expect a package? Thanks, Armin

**Comment by saveenr** [comment]
We'll be publishing that package later today. 

One note: for the time being, the WPF nuget package will have a slightly different NuGet package name. This is due to some process issues on our end. Eventually we'll resolve that issue also.

**Comment by juavol** [comment]
Any update on this? 

Last statement was that it should be published 8 days ago.

**Comment by jobubo** [comment]
@juavol: I assume @saveenr does this in his spare time. I would like to ask you for patience and rather be thankful for everything he has done so far. He does a great job and will publish the package as soon as they have time for this.

**Comment by juavol** [comment]
Agreed, this appears to be a great project and I am thankful for everyone contributing to it - especially if it is not their primary job. My apologies if my post did not correctly represent that.

However, an ETA would be good. I am fine if it takes 3 more days, 3 more weeks or we do not know when/if it will be available.

**Comment by paulovila** [comment]
Hi I published the WpfGraphControl https://www.nuget.org/packages/WpfGraphControl
